Support pilot and galley metrics (#3734)
DataDog/integrations-core: istio/README.md

### Configuration

#### Connect the Agent

Edit the `istio.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][2], to connect it to Istio. See the [sample istio.d/conf.yaml][3] for all available configuration options:

```
init_config:

instances:
  - istio_mesh_endpoint: http://istio-telemetry.istio-system:42422/metrics
    mixer_endpoint: http://istio-telemetry.istio-system:15014/metrics
    galley_endpoint: http://istio-galley.istio-system:15014/metrics
    pilot_endpoint: http://istio-pilot.istio-system:15014/metrics
    send_histograms_buckets: true
```

> The first two endpoints need to be connected to the check for it to work. To learn more about the prometheus adapter, you can read the [istio documentation][4].

Reviewer: ```suggestion
The first two endpoints are required for the check to work. See the [istio documentation][4] to learn more about the prometheus adapter.
```
